src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalAmount = 0;
+  const rentPerDay = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
```

Comment:
Don't hardcode some values that may change in the future in variable names (numbers etc.)
Something like basicDiscount and so on would be good.

Reply:
I agree with Denys, you should take a look at the checklist again